README.md

```diff
@@ -6,27 +6,392 @@
+
+[WSL2]: https://docs.microsoft.com/en-us/windows/wsl/install
+
+Requirements:
+
+- CPython (check `pyproject.toml` for version)
+- [Poetry] (latest version)
+- [PostgreSQL] (with the [PostGIS] extension) (version 13 or newer)
+- [Redis] (version 7 or newer)
+- [GDAL] (version compatible with Django, check their documentation for more info)
+  - Ubuntu: `sudo apt-get install gdal-bin`
+  - Mac: `brew install gdal`
+- [gettext]
+  - Ubuntu: `sudo apt-get install gettext`
+  - Mac: `brew install gettext`
+
+[Poetry]: https://python-poetry.org/
+[PostgreSQL]: https://www.postgresql.org/
+[PostGIS]: https://postgis.net/
+[Redis]: https://redis.io/
+[GDAL]: https://gdal.org/index.html
+[gettext]: https://www.gnu.org/software/gettext/
+
+> Installation instructions for dependencies will vary based on your OS and can
+> change over time, so please refer to the official documentation for each dependency
+> on how to set them up correctly.
+
+Now, follow the steps below in the `backend` directory.
+
+1. Copy `.env.example` to `.env`.
+
+```shell
+cp .env.example .env
+```
+
+> This file contains environment variables used by the project. You can modify these
+> to suit your local development environment.
+
+2. Copy `local_settings_example.py` to `local_settings.py`.
+
+```shell
+cp local_settings_example.py local_settings.py
+```
+
+> These can be used to modify settings for local development without changing the main settings file.
+
+3. Create a virtual environment & install dependencies.
+
+```shell
+poetry install
+```
+
+4. Add pre-commit hooks
+
+```shell
+poetry run pre-commit install
+```
+
+5. Run migrations
+
+```shell
+poetry run python manage.py migrate
+```
+
+6. Generate test data
+
+```shell
+poetry run python manage.py create_test_data
+```
+
+7. Start the server
+
+```shell
+poetry run manage.py runserver localhost:8000
+```
+
+Backend should now be running at `localhost:8000`.
+
+> Since the backend is not in the root of the project, the source path is not correct for linting out of the box.
+> Fixing this is specific to your IDE, but here are some setups for popular ones:
+>
+> PyCharm: Right click on the `backend` folder and select "Mark Directory as" and then select "Sources Root".
+>
+> VSCode: In your workspace settings (`.vscode/settings.json`), add the following:
+>
+> ```json
+> {
+>     "python.analysis.extraPaths": [
+>         "$(workspaceFolder)/backend"
+>     ]
+> }
+> ```
+
+### Linting
+
+It's recommended to set up [Ruff] linting and formatting support in your editor.
+
+[Ruff]: https://docs.astral.sh/ruff/editors/setup/
+
+### Testing
+
+Tests are run with `pytest`.
+
+Some flags that can save time when running tests:
+
+- To skip slow-running tests: `pytest --skip-slow`
+- To retain test database between runs: `pytest --reuse-db`
+- To skip migration-checks at the start of tests: `pytest --no-migrations`
+- To run tests in parallel: `pytest -n 8 --dist=loadscope` (=8 cores, use `-n auto` to use all available cores)
```

Comment:
`--dist=loadscope` Should not be needed anymore, but for some reason tests run ~1 sec faster with it enabled 😄 🤷

Reply:
Might as well include it then